### PR TITLE
add community, remote year

### PR DESCRIPTION
**Pull request by hyekyung2**
Add community, Remote Year (Traveling with techies, 75 people for 1 year)

### CODE REVIEW

**Comment by bzz** [comment]
Thank you @hyekyung2 for sharing!

**Comment by hyekyung2** [comment]
Add #Housing (Add Common in NY, Hive Arena in Seoul)

**Comment by bzz** [comment]
It might be better to add http://en.hivearena.com/ instead though

File: README.md [hunk 1]
```diff
@@ -156,6 +156,8 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   6. [Work From Curacao](http://www.workfromcuracao.com/) - Coliving space in Curacao
   7. [Working Retreat](http://workingretreat.com) - Coliving space in Thailand
   2. [Worldpackers](https://www.worldpackers.com) - Housing/work exchange
+  1. [Common] (http://www.hicommon.com/) - Coliving House in NewYork
```
Comment (RichardLitt) [inline README.md]: Can you alphabetize these?

**Comment by jdsharp** [comment]
@RichardLitt http://workingretreat.com domain has turned into a parking page

**Comment by RichardLitt** [comment]
@jdsharp Thanks. Removed. 

Merged in 3ed537f966e678c246037813c0dfcf05a829bdfd. Thank you!

**Pull request closed without merging**